### PR TITLE
Add workers module to root, fix compile

### PR DESCRIPTION
**Pull request by armanbilge**
CI is failing on main because mergify merged a bad PR.

### CODE REVIEW

**Review by toddburnside — APPROVED** [review]
LGTM

**Review by rpiaggio — APPROVED** [review]
LGTM

File: build.sbt [hunk 1]
```diff
@@ -83,7 +83,11 @@ lazy val workers = project
     libraryDependencies ++= LucumaCatalog.value ++
       Http4sDom.value ++
       Log4Cats.value ++
-      ScalaWebAppUtil.value
+      ScalaWebAppUtil.value,
+    Test / scalaJSLinkerConfig ~= {
+      import org.scalajs.linker.interface.OutputPatterns
+      _.withOutputPatterns(OutputPatterns.fromJSFile("%s.mjs"))
```
Comment (toddburnside) [inline build.sbt:89]: I'm afraid to ask, but: what does this do?

Comment (toddburnside) [reply build.sbt:89]: Oh. I should have guessed... Thanks for the info.

**Pull request merged**